radix-substate-store-impls/src/state_tree/types.rs

```diff
@@ -1169,7 +1169,7 @@ impl<K: Clone> From<LeafNode<K>> for Node<K> {
 
 impl<P: Clone> Node<P> {
     /// Creates the [`Internal`](Node::Internal) variant.
-    #[cfg(any(test, feature = "fuzzing"))]
+    #[cfg(test)]
```

Comment:
The `fuzzing` feature mention was a remanent from when we imported the code originally and as of 1.80 the compiler catches that this isn't a supported feature in the crate.